mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/proxy/PulsarServiceLookupHandler.java

```diff
@@ -47,78 +53,109 @@ public class PulsarServiceLookupHandler implements LookupHandler {
     private final PulsarClientImpl pulsarClient;
     private final MetadataCache<LocalBrokerData> localBrokerDataCache;
     private final PulsarService pulsarService;
+    private final MQTTProxyConfiguration proxyConfig;
+    private final ExecutorProvider executorProvider;
 
     public PulsarServiceLookupHandler(PulsarService pulsarService, MQTTProxyConfiguration proxyConfig) {
         this.pulsarService = pulsarService;
+        this.proxyConfig = proxyConfig;
+        this.executorProvider = new ExecutorProvider(proxyConfig.getLookupThreadPoolNum(), "pulsar-lookup-thread");
```

Comment:
Add mop prefix name would be great.

pulsar-lookup-thread >  mop-lookup-thread

Reply:
The point is May have other protocols handler also have this name.

Reply:
> The point is may have other protocols handler also have this name.

Yes, mop prefix is more appropriate